ansible/roles/terraform_config/templates/environment.tfvars.j2

```diff
@@ -12,38 +12,24 @@ admin_ips = "{{ admin_ips }}"
   public_cidr_block_default_c 
   ] | join(' ') -%}
 public_cidr_block = "{{ public_cidr_block | default(public_cidr_block_defaults) }}"
-{% set indexer_cidr_block_defaults = vpc_cidr_block | ipsubnet(27, 320) -%}
-indexer_cidr_block = "{{ indexer_cidr_block | default(indexer_cidr_block_defaults) }}"
 
-{% set presentation_cidr_block_default_a = vpc_cidr_block | ipsubnet(24, 21) -%}
-{% set presentation_cidr_block_default_b = vpc_cidr_block | ipsubnet(24, 22) -%}
-{% set presentation_cidr_block_default_c = vpc_cidr_block | ipsubnet(24, 23) -%}
-{% set presentation_cidr_block_defaults = [
-  presentation_cidr_block_default_a, 
-  presentation_cidr_block_default_b, 
-  presentation_cidr_block_default_c 
+{% set openregister_cidr_block_default_a = vpc_cidr_block | ipsubnet(24, 21) -%}
+{% set openregister_cidr_block_default_b = vpc_cidr_block | ipsubnet(24, 22) -%}
+{% set openregister_cidr_block_default_c = vpc_cidr_block | ipsubnet(24, 23) -%}
+{% set openregister_cidr_block_defaults = [
+  openregister_cidr_block_default_a, 
+  openregister_cidr_block_default_b, 
+  openregister_cidr_block_default_c 
   ] | join(' ') -%}
-presentation_cidr_block = "{{ presentation_cidr_block | default(presentation_cidr_block_defaults) }}"
-{% set presentation_database_cidr_block_default_a = vpc_cidr_block | ipsubnet(27, 160) -%}
-{% set presentation_database_cidr_block_default_b = vpc_cidr_block | ipsubnet(27, 161) -%}
-{% set presentation_database_cidr_block_default_c = vpc_cidr_block | ipsubnet(27, 162) -%}
-{% set presentation_database_cidr_block_defaults = [
-  presentation_database_cidr_block_default_a, 
-  presentation_database_cidr_block_default_b, 
-  presentation_database_cidr_block_default_c 
+openregister_cidr_block = "{{ openregister_cidr_block | default(openregister_cidr_block_defaults) }}"
+{% set openregister_database_cidr_block_default_a = vpc_cidr_block | ipsubnet(27, 160) -%}
+{% set openregister_database_cidr_block_default_b = vpc_cidr_block | ipsubnet(27, 161) -%}
+{% set openregister_database_cidr_block_default_c = vpc_cidr_block | ipsubnet(27, 162) -%}
```

Comment:
yeah, we should make all subnets a `/24` at least to avoid issues of running out of IPs like we did with the public cidr blocks before.